src/pages/password-entry/components/PasswordEntry.vue

```diff
@@ -1,8 +1,13 @@
 <script setup lang="ts">
 import { MasterForm } from '@/features/entry'
+import ModalWindow from '@/shared/ui/modal-window/modalWindow.vue'
+import { open } from '@/shared/lib/modal'
 </script>
 
 <template>
-  <h1>Password entry</h1>
-  <MasterForm />
+  <main :inert="open">
+    <h1>Password entry</h1>
+    <MasterForm />
+  </main>
+  <ModalWindow />
```

Comment:
Если один блок кода занимает несколько строк, то должны быть отступы в одну пустую строку до и после него.

```suggestion
  </main>

  <ModalWindow />
```